ui/components/MesheryPatterns.js

```diff
@@ -427,9 +427,11 @@ function MesheryPatterns({
           const rowData = patterns[tableMeta.rowIndex]
           return (
             <>
-              <IconButton onClick={() => setShowForm({ pattern : patterns[tableMeta.rowIndex], show : true })}>
-                <ListAltIcon />
-              </IconButton>
+              <Tooltip title = "configuration">
```

Comment:
```suggestion
              <Tooltip title = "Configure">
```
Action = verb

Thanks for this, @warunicorn19 👍

Reply:
🤦‍♂️yes, my bad.